site/content/contribute/server/dependencies.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Dependencies"
+date: 2019-03-27T16:00:00-0700
+weight: 5
+subsection: Server
+---
+
+
+The Mattermost server uses [go modules](https://github.com/golang/go/wiki/Modules) to manage dependencies. To manage dependencies you must have modules enabled.
+
+
+## Enabling go modules support (Do this first!)
+
+While golang migrates to support go modules, if you are developing in the GOPATH you need to set the environment variable `GO111MODULE` to `on` for example:
```

Comment:
Not sure if "to" is needed in the first sentence

Reply:
Seems required to me.